drivers/at86rf2xx/include/at86rf2xx_registers.h

```diff
@@ -19,6 +20,7 @@
  * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  * @author      Kaspar Schleiser <kaspar@schleiser.de>
  * @author      Kévin Roussel <Kevin.Roussel@inria.fr>
+ * @author      Sebastian Meiling <s@mlng.net>
```

Comment:
For this file it seems a little weird, that you just use this commit to add your own copyright ;-)

Reply:
mhm, weird it is ...